.eslintrc.cjs

```diff
@@ -14,6 +14,7 @@ module.exports = {
   plugins: ['react-refresh'],
   rules: {
     'react-refresh/only-export-components': ['warn', { allowConstantExport: true }],
+    '@typescript-eslint/no-misused-promises': 'off',
```

Comment:
why do you need to add this?

Reply:
we don't need eslint to check how promises are used because we have typescript itself which can do so for us. 
if i dont write this code i would have issue with `onSubmit={handleSubmit(onSubmit)}`
and this was suggested by one of our mentors

Reply:
i think this can be fixed: `onSubmit={(e) => void handleSubmit(onSubmit)(e)`